c/frame/py_frame.cc

```diff
@@ -100,6 +100,51 @@ oobj Frame::copy(const PKArgs&) {
 
 
 
+//------------------------------------------------------------------------------
+// export_names()
+//------------------------------------------------------------------------------
+
+static PKArgs args_export_names(
+  0, 0, 0, false, false,
+  {}, "export_names",
+
+R"(export_names(self)
+--
+
+Create local variables for each column of this frame.
+
+This method inserts into `globals()` as many new variables as there are
+columns in this frame. Each variable is an f-expression referring to
+its column. For example, if the frame has columns A, B, and C, then
+three new variables will be created A = f.A, B = f.B and C = f.C.
+After calling this method you will be able to write column expressions
+using the column names directly, without using the f symbol:
+
+    DT[A + B > C, :]
+
+If a variable with the same name's as one of the columns already
+exists in the globals, it will be overwritten. If the same variable
+exists locally, it will remain as-is.
+
+This method is effectively equivalent to
+
+    for name in self.names:
+        globals()[name] = f[name]
+)");
+
+void Frame::export_names(const PKArgs&) {
+  py::oobj f = py::oobj::import("datatable", "f");
+  py::odict globals = py::robj(PyEval_GetGlobals()).to_pydict();
+  py::otuple names = dt->get_pynames();
+  for (size_t i = 0; i < dt->ncols; ++i) {
+    py::robj name = names[i];
+    globals.set(name, f.get_item(name));
```

Comment:
I suppose this will overwrite the variable with the same name that already exists in the global scope? May be we should mention this in the documentation.

Reply:
It is mentioned in line 125 above:
> If a variable with the same name's as one of the columns already
exists in the globals, it will be overwritten. If the same variable
exists locally, it will remain as-is.